js/loader.js

```diff
@@ -35,7 +35,7 @@ var Loader = (function() {
 				// This is done after all the moduels so we can
 				// overwrite all the defined styls.
 
-				loadFile("css/custom.css", function() {
+				loadFile(config.customCss, function() {
```

Comment:
@roramirez @JanLoebel The mirror will most likely freeze up if it attempts to load a non-existent file from C++ (the underlying language in Node). We should have a try-catch case to figure out the status or use `fs.exists()`.